docs/low-code-sql/development/models.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Models
+id: models
+description: Models are a sequence of data transformation steps which define a single table or view
+sidebar_position: 1
+tags:
+  - model
+  - SQL
+  - table
+---
+
+:::caution
+This detailed page all about SQL Models is under construction. Please pardon our dust.
```

Comment:
Lineage view is at the project level, so put the project lineage view on the development page. Walkthrough the layout of the Project page, (browser, fabric, commit footer, visual and code view, mention config will be discussed in a separate page.]

The model page should mention the project browser.  Mention Interactive execution, git footer. We don't have dependencies as a category on the project browser, I'll file a bug for this. If there are no dependencies then the browser should show a list of zero.